### ISSUE TITLE
[feature] animation toggle

### ISSUE THREAD

**Issue by qube-03** [OlauncherCF/OlauncherCF#291]
hi! im currently using this as my main launcher and everytime i get bored and try a new one i always end up coming back to this one. no fiddling around with icons, no distractions and a great way of searching for apps.

one thing i think would be cool to add is a toggle in the settings for turning off the app opening/closing animations. its a bit hard to explain, all i would like the toggle to do is to instantly open/close the actual app without showing the "default" animation, with the app icon showing for like a third of a second and then the content loading. closing/clicking on the home button would just cut you straight to the main screen.

im not sure how feasable this is, considering that apps need to load lmao and idk how app opening and closing works/if you even specify an animation or if its handled by the os.

still this would be cool, thanks!

tldr: add toggle in settings that turns off all animations pls
(a gif example can be provided if needed)

**Comment by jooooscha** [comment]
We had an issue about animations here: #11. It's a bit different, but I think the overall conclusion is the same.

Although animations can be handled by the launcher (I think?), I don't think this is feasible for this project. This launcher is not only minimal for the user, but also minimal for the developer.

If someone has a simple implementation, I am happy to accept a PR. However, I don't think I will implement this in the near future.

Closing, for now.

**Comment by qube-03** [comment]
okay, thank you for the fast response and explanation!
